yarn.lock

```diff
@@ -27,7 +25,7 @@ abbrev@1:
   version "1.1.0"
   resolved "https://registry.yarnpkg.com/abbrev/-/abbrev-1.1.0.tgz#d0554c2256636e2f56e7c2e5ad183f859428d81f"
 
-accepts@1.3.3, accepts@~1.3.3:
+accepts@~1.3.3, accepts@1.3.3:
```

Comment:
Generally I think the rule is you shouldn't check in `yarn.lock` unless you've added new frontend dependencies. Right @attekei @tlrobinson @kdoh ? Becaus it seems like it make random meaningless changes every time a different person runs `yarn install`